README.md

```diff
@@ -1,5 +1,9 @@
 <p style="text-align:center;" align="center">
-  <img align="center" src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/layer5-no-trim.svg" alt="Layer5-logo" width="45%" />
+   <picture align="center">
+      <source media="(prefers-color-scheme: dark)" srcset="./src/assets/images/layer5/layer5-only/svg/layer5-light-no-trim.svg">
```

Comment:
Preferably, make a copy of this image under the `.github/assets/images/layer5`, so that there is no interdependency.

Reply:
Sure. On it!